{"name": "test_register[Create user - Firstname Numeric]", "status": "failed", "statusDetails": {"message": "AssertionError: FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:41 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1", "trace": "FAILURE: \u001b[31massert 200 == 400\ntests\\test_api.py:41 in test_register() -> check.equal(response.status_code,\n\u001b[39m\n------------------------------------------------------------\nFailed Checks: 1"}, "attachments": [{"name": "log", "source": "b0816c53-ff43-4b00-add3-201daafe5794-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case", "value": "{'Case': 'Create user - Firstname Numeric', 'Test Type': 'Negative', 'Request_Body': {'firstName': '18639552165331142846755624', 'lastName': 'ZkIrsWTLMSQkrMiTqLGjEhLDJf', 'username': 'jjrs7931', 'password': 'WJ575'}}"}], "start": 1670168718882, "stop": 1670168719704, "uuid": "31f12a1d-7a4e-45fb-9542-f7f6cedf5f41", "historyId": "8d4cf15c7d7b1e34fc6b9525a0de9107", "testCaseId": "bd51a8de3249e024de618e4e17a0a343", "fullName": "tests.test_api.TestApi#test_register", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestApi"}, {"name": "host", "value": "DESKTOP-EGG762M"}, {"name": "thread", "value": "11972-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api"}]}